server/views/pages/appointments/create-and-edit/category.njk

```diff
@@ -7,7 +7,11 @@
 
 {% set pageTitle = appointmentJourneyTitle("Type", session.appointmentJourney) %}
 {% set pageId = 'appointments-create-category-page' %}
-{% set backLinkHref = "/appointments/create/select-prisoner?query=" + session.appointmentJourney.prisoners[0].number %}
+{% if session.appointmentJourney.type == AppointmentType.BULK %}
+    {% set backLinkHref = "/appointments/create/review-prisoners" %}
+{% else %}
+    {% set backLinkHref = "/appointments/create/select-prisoner?query=" + session.appointmentJourney.prisoners[0].number %}
```

Comment:
This is only valid for individual I think so may be better to switch the if statement to if individual, this back link otherwise the one above